types/express-serve-static-core/index.d.ts

```diff
@@ -440,7 +440,10 @@ interface Request extends http.IncomingMessage, Express.Request {
 
     method: string;
 
-    params: any;
+    params: {
+        [namedParameter: string]: string;
+        [captureGroup: number]: string;
+    };
```

Comment:
I thought we wouldn't need the `captureGroup: number` signature, because all object keys are strings?